test/integration/client-side-encryption/client_side_encryption.spec.test.ts

```diff
@@ -81,8 +81,16 @@ describe('Client Side Encryption (Legacy)', function () {
 });
 
 describe('Client Side Encryption (Unified)', function () {
-  if (process.version.includes('18')) {
-    setDefaultResultOrder('ipv4first');
-  }
+  before(function () {
+    if (process.version.includes('18')) {
```

Comment:
16.17 is the current minor so this is going to run on 16.18 when it comes out, let's make sure this is running for the major 18 only